pkg/cluster/config.go

```diff
@@ -156,6 +195,34 @@ func (c *config) Validate() error {
 	return nil
 }
 
+// checkOpenShiftVersion checks the OpenShift cluster version and returns an error if it is lesser than
+// minOpenShiftVersion
+func checkOpenShiftVersion(openShiftVersion string) error {
+	if len(openShiftVersion) == 0 {
+		return fmt.Errorf("cluster version not set")
+	}
```

Comment:
Instead of doing this check, it would be more comprehensive to add the v prefix and then use [semver.isValid ](https://pkg.go.dev/golang.org/x/mod/semver#IsValid) to check that the version is formatted properly.

Alternatively you could do this within `getOpenShiftVersion`, renaming it to `getOpenShiftSemver`. That way it is guaranteed that `getOpenShiftSemver` is returning a valid semver, and the code within this function doesn't need to be as defensive.